app/helpers/search_helper.rb

```diff
@@ -9,4 +9,12 @@ def will_paginate_petitions(collection_or_options = nil, options = {})
        <span class='paginate paginate-next'>Next</span>"
     will_paginate *[collection_or_options, options].compact
   end
+
+  def filtered_petition_count(petitions)
+    if petitions.search?
+      pluralize(petitions.total_entries, 'result')
+    else
+      pluralize(petitions.total_entries, 'petition')
```

Comment:
Probably want a `number_with_delimiter` around the number (so we get `1,000,000` instead of `1000000`).